src/extras/core/Path.js

```diff
@@ -151,7 +151,8 @@ THREE.Path.prototype = Object.assign( Object.create( THREE.CurvePath.prototype )
 			xRadius, yRadius,
 			aStartAngle, aEndAngle,
 			aClockwise,
-			aRotation || 0 // aRotation is optional.
+			aRotation || 0, // aRotation is optional.
+			this.curves.length
```

Comment:
>I needed to store the index of the ellipse in curves somewhere... So I added the index of the ellipse in args

That kind of change is likely not to be accepted.

Do not worry about performance. A proper refactoring can be done later.